internal/ui/logger.go

```diff
@@ -1,12 +1,25 @@
 package ui
 
 import (
+	"github.com/dream11/odin/pkg/constant"
 	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
 )
 
 func init() {
+
 	log.SetFormatter(&log.TextFormatter{
-		ForceColors:   true,
-		DisableColors: false,
+		ForceColors:     true,
+		DisableColors:   false,
+		TimestampFormat: "2006-01-02 15:04:05", // Custom format
+		FullTimestamp:   true,
 	})
+	level, err := log.ParseLevel(strings.ToLower(os.Getenv(constant.LogLevelKey)))
+	if err != nil {
+		log.SetLevel(level)
+	} else {
+		// log error message
+		log.Error("Invalid log level")
```

Comment:
`log.error` will not exit the command, use `log.Fatal`

Reply:
Setting incorrect log levels should not impact command flow. Maybe a warning message

Reply:
Yeah a warning message with fallback to `info` works

Reply:
done